src/main/java/org/jnode/fs/xfs/extent/DataExtent.java

```diff
@@ -43,11 +43,13 @@
     public DataExtent(byte[] data, int offset) {
         super(data, offset);
 
-        blockCount = (int) (getUInt32(0xc) & 0x3ffffL);
-
-        startBlock = ((getUInt32(0xc) & 0xfffc0000L) >> 21) | (getUInt32(0x8) << 14) | ((getUInt32(0x4) & 0x3fL) << 38);
-
-        startOffset = ((getUInt32(0x0) & 0x7fffffff) << 22) | ((getUInt32(0x4) & 0xfffffcL) >> 9);
+        long valueUpper128bit = getInt64( 0);
```

Comment:
The Nuix Confluence documentation on code style mentions adhering to the Oracle code conventions https://www.oracle.com/java/technologies/javase/codeconventions-contents.html . Please adjust the spacing to follow these conventions.